model/articles.model.js

```diff
@@ -23,4 +23,25 @@ exports.selectArticles = () => {
     .then((results) => {
     return results.rows
     })
+}
+
+exports.updateArticleVotes = (article_id, inc_votes) => {
+  const { 'inc_votes' : newVote } = inc_votes;
+  if(newVote && typeof newVote !== 'number') {
+    return Promise.reject({status: 400, msg: 'Bad request'})
```

Comment:
You can actually use PSQL to help handle this as it will throw an error if you pass it anything other than a number.